hypothesis-python/src/hypothesis/core.py

```diff
@@ -39,12 +39,12 @@
 from coverage.collector import Collector
 
 import hypothesis.strategies as st
-from hypothesis import __version__
 from hypothesis.errors import Flaky, Timeout, NoSuchExample, \
     Unsatisfiable, DidNotReproduce, InvalidArgument, DeadlineExceeded, \
     MultipleFailures, FailedHealthCheck, HypothesisWarning, \
     UnsatisfiedAssumption, HypothesisDeprecationWarning
 from hypothesis.control import BuildContext
+from hypothesis.version import __version__
```

Comment:
Why did this move?

Reply:
It's now imported from `hypothesis.version` instead of from `hypothesis`, again to keep Mypy happy.  I suspect something about import order or ignoring dunder names in `__all__`?

Reply:
Oh! So it is. Right. That's weird but at least within normal WTF mypy bounds of weird...